src/docs/integration/linux/build-tools/buildroot.md

```diff
@@ -0,0 +1,80 @@
+# Buildroot
+
+You can integrate the Peridio Agent into buildroot by leveraging the [buildroot external tree](https://buildroot.org/downloads/manual/manual.html#outside-br-custom).
+
+Checkout the peridio external tree from GitHub: [peridio/buildroot-external-peridio](https://github.com/peridio/buildroot-external-peridio)
+
+Pass the location to the Peridio external tree when configuring the build directory from with the Buildroot tree, you can validate the external Peridio Agent build options are present through `menuconfig` or `xconfig`.
+
+```shell
+make BR2_EXTERNAL=/path/to/buildroot-external-peridio menuconfig
+```
+
+<img src="/img/integration/linux/buildroot-menuconfig-external.png"  />
+
+## Configuring peridiod
+
+The `peridiod` package in `buildroot-external-peridio` will install a systemd service and configure the location of the [peridiod configuration file](/integration/linux/peridiod#configuring) to the path `/etc/peridiod/peridio.json`. This file can be included as part of the inclding a overlay directory path to `BR2_ROOTFS_OVERLAY`. An example of this implementation can be found in the buildroot-external-peridio repo with the included QEmu target.
+
+## Building FWUP archives
+
+A `fwup` configuration file can be included in your buildroot tree and configured to produce a fwup archive as part of the build process. This fwup archive can then be signed and uploaded to peridio for further distribution.
```

Comment:
When writing in-line about fwup you write it as

> `fwup` 6 times

and

> fwup 11 times

IMO we should pick one and be consistent - I'd lean towards fwup.